.github/workflows/qemu.yml

```diff
@@ -102,6 +102,13 @@ jobs:
           echo "\n ### Testing ${config_name} ###"
           umf/scripts/qemu/start_qemu.sh ${config_name}
 
-          ssh testuser@127.0.0.1 -p 2222 -t "bash /home/testuser/scripts/qemu/run-tests.sh"
+          ssh testuser@127.0.0.1 -p 2222 -t "bash /home/testuser/scripts/qemu/run-tests.sh COVERAGE ${config_name}"
+          scp -r -P 2222 testuser@127.0.0.1:/home/testuser/coverage ./
           ssh testuser@127.0.0.1 -p 2222 -t "sudo shutdown -h now"
         done
+        ls -al ./coverage
```

Comment:
this `ls` seems redundant, we print content of the coverage dir at the end of each `qemu/run-test.sh`

Reply:
I removed the other one. It is most convenient to check it at the end of this huge log file.

Done